framework/src/meshgenerators/PatternedMeshGenerator.C

```diff
@@ -183,6 +220,74 @@ PatternedMeshGenerator::generate()
         *_row_meshes[i], bottom, top, TOLERANCE, /*clear_stitched_boundary_ids=*/true);
   }
 
+  // Set outer boundary sidesets and nodesets
+  BoundaryInfo & boundary_info = _row_meshes[0]->get_boundary_info();
+
+  // Redefine boundary ids for output mesh
+  boundary_names = {
+      isParamValid("new_left_boundary") ? getParam<BoundaryName>("new_left_boundary")
+                                        : getParam<BoundaryName>("left_boundary"),
+      isParamValid("new_right_boundary") ? getParam<BoundaryName>("new_right_boundary")
+                                         : getParam<BoundaryName>("right_boundary"),
+      isParamValid("new_top_boundary") ? getParam<BoundaryName>("new_top_boundary")
+                                       : getParam<BoundaryName>("top_boundary"),
+      isParamValid("new_bottom_boundary") ? getParam<BoundaryName>("new_bottom_boundary")
+                                          : getParam<BoundaryName>("bottom_boundary")};
```

Comment:
this default fall back should be documented

Reply:
This is documented in the PatternedMeshGenerator markdown file now